docs/index.pug

```diff
@@ -0,0 +1,11 @@
+extends /layout/base.pug
+
+block content
+	main.col-md-24
+		h3 CKEditor 5 Documentation
+		hr
```

Comment:
This `<hr>` is absolutely incorrect here. If you want a line below a heading, then use CSS.